playground/my_playground.py

```diff
@@ -0,0 +1,99 @@
+
+    Args:
+        filename (string): a csv filename 
+    """
+    dataframe = pd.read_csv(filename)
+    print('Printing dataframe...')
+    print(dataframe)
+    return dataframe
+    
+def df_iter_rows(df):
+    """(dataframe.iterrows) uses dataframe.iterrows to print first 3 rows in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    print('(ITER_ROWS) Printing first 3 rows in the dataframe...')
+    for index, row in df.iterrows():
+        if index == 3: break
+        print('ROW:', index)
+        print(row)
+    
+def df_iter_tuples(df):
+    """(dataframe.itertuples) uses dataframe.itertuples to print first 3 row tuples in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    index = 0
+    print('(ITER_TUPLES) Printing first 3 tuples in the dataframe...')
+    for tuple in df.itertuples():
+        if index == 3: break
+        print('TUPLE:', index)
+        print(tuple)
+        index = index + 1 
+
+def df_iter_columns(df):
+    """converts df into a python list to print first 3 columns in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    columns = list(df)
+    index = 0
+    print('(COLUMNS) Printing first 3 columns...')
+    for i in columns:
+        if index == 3: break
+        print('COLUMN:', index)
+        print (df[i])
+        index = index + 1
+
+def load_file_into_db(filename, table, conn):
+    """(dataframe.to_sql) load a table into a database connection istance
+
+    Args:
+        filename (string): name of the file
+        table (string): name of the table
+        conn (db): connection to sqlite database instance 
+    """
+    dataframe = pd.read_csv(filename)
+    dataframe.to_sql(name=table, con=conn, if_exists="replace")
+
+def preview_data_in_db(table, conn):
+    """(pd.read_sql) read table contents in a database connection istance to print first 10 rows
+
+    Args:
+        table (string): name of the table
+        conn (db): connection to sqlite database instance 
+    """
+    sql = f"SELECT * FROM '{table}' limit 10;"
+    dataframe = pd.read_sql(sql=sql, con=conn)
+    pprint(dataframe)
+
+if __name__ == "__main__":
+    source_file = "../data/sample_data.csv"
+    table_name = "sample_table"
+    conn = sqlite3.connect("my_local.db")
+    if (sys.argv[1] == 'df-read'):
+        df = read_data(filename=source_file)
+    elif (sys.argv[1] == 'df-iterrows'):
+        df = read_data(filename=source_file)
+        df_iter_rows(df)
+    elif (sys.argv[1] == 'df-itertuples'):
+        df = read_data(filename=source_file)
+        df_iter_tuples(df)
+    elif (sys.argv[1] == 'df-itercolumns'):
+        df = read_data(filename=source_file)
+        df_iter_columns(df)
+    elif (sys.argv[1] == 'db-load'):
+        load_file_into_db(source_file, table_name, conn)
+    elif (sys.argv[1] == 'db-load-read'):
+        load_file_into_db(source_file, table_name, conn)
+        preview_data_in_db(table_name, conn)
```

Comment:
It may help with [code dryness](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to leverage [click](https://pypi.org/project/click/) here.

The `@click` is called a [python decorator](https://www.freecodecamp.org/news/python-decorators-explained-with-examples/#:~:text=You'll%20use%20a%20decorator,same%20code%20on%20multiple%20functions).

For example, you could have a flag called `--command` that you pass to a function:
```
import click

@click.command(help="""Example: python3 playground/my_playground.py --command 'df-read'

Choose one of the following options:
- df-read
- df-iterrows
- df-itertuples
- ...
"""
)
@click.option("--command", default="db-load-read")
def run(command):
      if (command == 'df-read'):
        df = read_data(filename=source_file)
      ...
    elif (command == 'db-load-read'):
        load_file_into_db(source_file, table_name, conn)
        preview_data_in_db(table_name, conn)
    else:
        print("Invalid input detected!!!!")
        with click.Context(command) as ctx:
               click.echo(command.get_help(ctx))
```
([help message](https://stackoverflow.com/questions/43140639/does-click-lib-provide-a-way-to-print-the-builtin-help-message))

Reply:
click should give you ability to do everything from `run.sh` in the same part of python code where its being passed via `sys.argv`


try using [this line of code](https://github.com/org-not-included/mlb-stats/blob/26c7b5fdbe30fe80ba0bffb7400625f45302a591/backend/util/cli_helpers.py#L19) in your `if name =main` block